controllers/protectedvolumereplicationgrouplist_controller.go

```diff
@@ -50,31 +50,33 @@ type ProtectedVolumeReplicationGroupListInstance struct {
 // - https://pkg.go.dev/sigs.k8s.io/controller-runtime@v0.9.2/pkg/reconcile
 func (r *ProtectedVolumeReplicationGroupListReconciler) Reconcile(ctx context.Context, req ctrl.Request,
 ) (ctrl.Result, error) {
-	log := ctrl.Log.WithName("controllers").WithName("protectedvolumereplicationgrouplist").
-		WithValues("name", req.NamespacedName.Name)
-
-	log.Info("protectedvolumereplicationgrouplist reconcile start")
-
 	// save all the commonly used parameters in a struct
 	s := ProtectedVolumeReplicationGroupListInstance{
 		reconciler: r,
 		ctx:        ctx,
-		log:        log,
+		log:        ctrl.Log.WithName("pvrgl").WithValues("name", req.NamespacedName.Name),
```

Comment:
Why we don't care about the namespace?

Reply:
Good point. I think we did not log the namespace because we started with DRPolicy as the example and that is always in the Ramen namespace. All the other resources must log the namespace too. I will update it.

Reply:
I take that back. ProtectedVolumeReplicationGroupList is cluster scoped. No namespace to log.